references/shacl-20170720.ttl

```diff
@@ -545,7 +545,7 @@ sh:DatatypeConstraintComponent
 sh:DatatypeConstraintComponent-datatype
 	a sh:Parameter ;
 	sh:path sh:datatype ;
-	sh:nodeKind sh:IRI ;
+	sh:datatype xsd:anyURI ;
```

Comment:
```suggestion
	sh:nodeKind sh:IRI ;
```

I am not sure if this change is correct. sh:IRI is actually semantically correct IMHO. CodeGen should support that, or not?

Reply:
Please also verify and possibly update all other places. This change does not make sense to me and CodeGen should support sh:IRIs in general. I will not add GitHub comments to each individual change below, because there are too many.